core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
 
 # 2 Design and Architecture
 
+This section provides definitions, key concepts, and overviews of the components of a Baseline Protocol Implementation compliant with the requirements of this document. 
+
+## 2.1 Agreement
+An Agreement between two or more Counterparties allows for transactions between them dealing with, for example, commercial items such as products. An Agreement governs and defines all transactions between counterparties.
+
+**[R1]**	Transacting counterparties  MUST have a corresponding agreement.
+
+## 2.2	State Object
+A State Object is an item which can be exchanged between counterparties to an agreement, whose state the counterparties have agreed on and which is defined in an agreement. In the context of this standard, a State Object is assumed to represent either an asset, a product offering or a service offering transacted between counterparties.
+Examples include but are not limited to:
+* Digital Services
+* Physical Products
+* Financial Assets
+ 
+## 2.3 Transacting Counterparties
```

Comment:
'state objects' is sometimes capitalised and sometimes not, would be good to use the same standard form.

Reply:
@timhc22 capitalized everything and will push new commit